deltachat-server/rpc-client.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+import asyncio
+import json
+
+
+class Deltachat:
+    def __init__(self, process):
+        self.process = process
+        self.event_queue = asyncio.Queue()
+        self.id = 0
+        self.reader_task = asyncio.create_task(self.reader_loop())
+
+        # Map from request ID to asyncio.Event.
+        # When event happens, it is replaced with a response.
+        self.request_events = {}
+
+    async def reader_loop(self):
+        while True:
+            line = await self.process.stdout.readline()
+            response = json.loads(line)
+            if "id" in response:
+                response_id = response["id"]
+                event = self.request_events[response_id]
+                self.request_events[response_id] = response
+
+                # Notify that response arrived.
+                event.set()
+            else:
+                if response["method"] == "event":
+                    # An event notification.
+                    await self.event_queue.put(response["params"])
+                continue
```

Comment:
bit of a lost `continue`, seems out of place given it's not in the other branch as well.